2200048/문제4.py

```diff
@@ -0,0 +1,10 @@
+T = int(input())
+for i in range(1,T + 1):
+    n_avg = 0
+    n = int(input())
+    inc = list(map(int,input().split()))
+    avg_n = int(sum(inc) / n)
```

Comment:
깔끔한 풀이네요! 👍 
```suggestion
    avg_n = sum(inc) / n
```
정확한 평균값은 `int` 함수 없이 구하는 게 맞아요! (이 문제에서는 정답에 영향을 끼치지 않습니다 😄 )